From bfa53e1c67eddb14887cda37d7c89c76f4cce378 Mon Sep 17 00:00:00 2001
From: Ralf Nolden <nolden@kde.org>
Date: Tue, 10 May 2016 16:34:37 +0200
Subject: [PATCH] Compile fix: remove _POSIX_C_SOURCE usage

Remove _POSIX_C_SOURCE usage as the reason why it was added is not
clear anymore and it causes compile errors on BSD systems if not
circumvented by adding further defines to re-enable function calls
hidden by the _POSIX_C_SOURCE define. (__BSD_VISIBLE on FreeBSD/OpenBSD
and _NETBSD_SOURCE on NetBSD)

Change-Id: Ic6b49ddcd6c481b0f2acd598cea5470604e00507
Reviewed-by: Thiago Macieira <thiago.macieira@intel.com>
--- src/corelib/tools/qelapsedtimer_unix.cpp.orig	2016-02-24 20:16:34 UTC
+++ src/corelib/tools/qelapsedtimer_unix.cpp
@@ -31,9 +31,6 @@
 **
 ****************************************************************************/
 
-// ask for the latest POSIX, just in case
-#define _POSIX_C_SOURCE 200809L
-
 #include "qelapsedtimer.h"
 #if defined(Q_OS_VXWORKS)
 #include "qfunctions_vxworks.h"
